### ISSUE TITLE
Training base model appears to yield higher error rate than the checked-in model

### ISSUE THREAD

**Issue by prpaxson** [githubharald/SimpleHTR#44]
Hello Harald,

I am trying to train the model with some new IAM data, but before doing that I tried training with just the base dataset I was directed to in the README. The accuracy of the model changed from ~10% error rate in the GitHub model here, to ~43% error rate (according to accuracy.txt). 

I ran into similar problems on:
* Tensorflow 1.10, Python 3.6.5, Ubuntu 16.04.5 LTS
* Tensorflow 1.10, Python 3.7.2, MacOS 10.13

I got this high error rate just by following the training instructions in the README. Hope to hear back from you soon on how you produced a more accurate model.

Thanks for your great work!

**Comment by githubharald** [comment]
Hi,

just to be sure - did you remove the pre-trained model (model/checkpoint and model/snapshot*) before starting the training? If not, please remove it and and train from scratch.

**Comment by prpaxson** [comment]
Yes, I removed all files in the model folder before starting.

**Comment by githubharald** [comment]
The last model I trained myself was with the neural network architecture at commit 35ddc4873d8ea2fad093692a0b51ffd0d045997f
Please checkout this commit and see if training converges to ~13% character error rate.
Just want to be sure that it's not an issue of batch normalization which was added afterwards.

**Comment by prpaxson** [comment]
The first time I ran into this issue was in September, on commit 7388432 (on the MacOS run). I can run it again with the commit you mentioned, but it will take a few hours.

**Comment by githubharald** [comment]
This was before batch normalization, so it's not caused by it.
Did you check the data - are there ~115k words images?
Sorry, but that' all I can think of at the moment. I never had problems training it.

**Comment by prpaxson** [comment]
A line count of the words.txt file returned 115338 lines.

checkDirs.py returned OK for all of the tests. 

I will comment the results of the training on commit 35ddc48 once it's done. Let me know if there is anything else I can do to try and troubleshoot this.

**Comment by prpaxson** [comment]
Training has stopped. Here is the report:
Character error rate: 14.518935%. Word accuracy: 66.400000%.

The error rate went up, but less significantly than other times. Is this an acceptable error rate change?

**Comment by prpaxson** [comment]
Rerunning on the latest version yielded similar small error increases. I will reopen this if it begins to have large error rates with modified versions of the imagesets.
